hub/constraints/instruction-handling/halting/return.lisp

```diff
@@ -266,38 +266,38 @@
-                                                                                ;; tgt_offset_lo                       ;; target offset low
-                                                                                ;; size                                ;; size
-                                                                                ;; ref_offset                          ;; reference offset
-                                                                                ;; ref_size                            ;; reference size
-                                                                                (return-instruction---exception-flag-ICPX)   ;; success bit
-                                                                                ;; limb_1                              ;; limb 1
-                                                                                ;; limb_2                              ;; limb 2
-                                                                                ;; exo_sum                             ;; weighted exogenous module flag sum
-                                                                                ;; phase                               ;; phase
+                                  (set-MMU-instruction---invalid-code-prefix    ROFF_RETURN___1ST_MISC_ROW                        ;; offset
+                                                                                CONTEXT_NUMBER                                    ;; source ID
+                                                                                ;; tgt_id                                         ;; target ID
+                                                                                ;; aux_id                                         ;; auxiliary ID
+                                                                                ;; src_offset_hi                                  ;; source offset high
+                                                                                (return-instruction---offset-lo)                  ;; source offset low
+                                                                                ;; tgt_offset_lo                                  ;; target offset low
+                                                                                ;; size                                           ;; size
+                                                                                ;; ref_offset                                     ;; reference offset
+                                                                                ;; ref_size                                       ;; reference size
+                                                                                (- 1 (return-instruction---exception-flag-ICPX))  ;; success bit; this double negation stuff will be resolved by spec issue #715
```

Comment:
This was a bug

Reply:
shouldn't the definition of `set-MMU-instruction---invalid-code-prefix` change too?

Reply:
No. The issue was that we were not respecting the HUB/MMU interface. We were feeding the "failure bit" in the HUB and expecting the "success bit" in the MMU.